pkg/storaged/dialog.jsx

```diff
@@ -1109,6 +1109,7 @@ export const BlockingMessage = (usage) => {
         vdo: _("backing device for VDO device"),
         "stratis-pool-member": _("member of Stratis pool"),
         mounted: _("Filesystem outside the target"),
+        "btrfs-device": _("device of btrfs volume"),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19690-20240111-133844-24a81545-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)